lib/spack/spack/installer.py

```diff
@@ -2481,7 +2649,7 @@ def install(self):
         """
         try:
             with fs.replace_directory_transaction(self.task.pkg.prefix):
-                self.installer._install_task(self.task, self.install_status)
+                return self.installer._install_task(self.task, self.install_status)
```

Comment:
This method does not return a value nor does it appear to need to return at this point.